src/test/java/hudson/plugins/parameterizedtrigger/test/FileBuildParameterFactoryTest.java

```diff
@@ -245,22 +257,31 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
             }
         });
 
-        // add Trigger builder, with file parameter factory
-        projectA.getBuildersList().add(createTriggerBuilder(projectB, NoFilesFoundEnum.SKIP, "UTF-8"));
+        // SECURITY-170: need to allow multibyte params that can't be traditionally declared.
```

Comment:
Why not? Is your IDE not honoring the POM declaration of UTF-8 sources or something?